openmmtools/multistate/multistatereporter.py

```diff
@@ -1307,6 +1309,29 @@ def write_online_data_dynamic_and_static(self, iteration: int, **kwargs):
         self.write_online_analysis_data(None, **kwargs)
         self.write_online_analysis_data(iteration, **kwargs)
 
+    def write_current_statistics(self, data):
+        """
+        Write real time YAML file with analysis data.
+
+        Overwrites file if it already exists.
+
+        Parameters
+        ----------
+        data: dict
+            Dictionary with the key, value pairs to store in YAML format.
+        """
+        reporter_dir, _ = os.path.split(self._storage_analysis_file_path)
+        output_filepath = f"{reporter_dir}/real_time_analysis.yaml"
```

Comment:
Can you list the path of the YAML file that will be generated in the docstring as well?